Library/Homebrew/cmd/update-report.rb

```diff
@@ -232,11 +232,11 @@ def migrate_legacy_repository_if_necessary
     end
 
     if (HOMEBREW_REPOSITORY/"Caskroom").exist?
-      FileUtils.cp_r "#{HOMEBREW_REPOSITORY}/Caskroom", "#{new_homebrew_repository}/Caskroom"
+      FileUtils.mv "#{HOMEBREW_REPOSITORY}/Caskroom", "#{HOMEBREW_PREFIX}/Caskroom"
```

Comment:
This is only run if `HOMEBREW_REPOSITORY` and `HOMEBREW_PREFIX` are both `/usr/local` so it's a no-op and can be removed.